tensorflow/g3doc/tutorials/tflearn/index.md

```diff
@@ -35,11 +35,13 @@ IRIS_TRAINING = "iris_training.csv"
 IRIS_TEST = "iris_test.csv"
 
 # Load datasets.
-training_set = tf.contrib.learn.datasets.base.load_csv(filename=IRIS_TRAINING,
-                                                       target_dtype=np.int)
-test_set = tf.contrib.learn.datasets.base.load_csv(filename=IRIS_TEST,
-                                                   target_dtype=np.int)
-
+training_set = tf.contrib.learn.datasets.base.load_csv_with_header(filename=IRIS_TRAINING, 
+                                                       target_dtype=np.int,
+                                                       features_dtype=np.float32)
```

Comment:
from the old load_csv -> load_csv_with_header translation seems like this should perhaps be np.float64?

https://github.com/tensorflow/tensorflow/commit/2d4267507e312007a062a90df37997bca8019cfb#diff-8e7d29906b35612ebe283482505429d4